frontend/packages/ux-editor/src/containers/DesignView/PageAccordion/PageAccordion.tsx

```diff
@@ -41,6 +42,7 @@ export const PageAccordion = ({
   onClick,
   pageIsReceipt,
   isValid,
+  areUniqueIds,
```

Comment:
Nitpicking, but maybe rename this to `areIdsUnique` or `hasUniqueIds`?

Reply:
fixed